csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt

```diff
@@ -38,7 +38,7 @@ creationDate:IGNORE|:START_ID(Forum)|:END_ID(Post)
 sf-003/parquet/9/part.0.parquet
 sf-003/parquet/9/part.1.parquet
 
-creationDate:IGNORE|:START_ID(Forum)|:END_ID(Person)
+creationDate:DATETIME|:START_ID(Forum)|:END_ID(Person)
```

Comment:
so we are not ignoring this any more?  Do you know why we ignored it in the first place?

Reply:
No clue why we were ignoring it initially (@tylershunt might know), but we're not ignoring it anymore since creationDate exists as a property of hasMember (an edge type between Forum and Person)

Reply:
Can we have comments in this file, so that we can provide more details and links to appropriate places in the spec?

Reply:
if this is indeed `hasMember` then I confirm this is correct to use DATETIME. 

By the way, did you check that what you have here matches our csv files?